assets/src/components/activities/directed-discussion/discussion/DirectedDiscussion.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useMemo } from 'react';
+//import { HintsDeliveryConnected } from 'components/activities/common/hints/delivery/HintsDeliveryConnected';
+import { StemDeliveryConnected } from 'components/activities/common/stem/delivery/StemDelivery';
+//import { castPartId } from '../../common/utils';
+import { DirectedDiscussionActivitySchema } from '../schema';
+import { DiscussionParticipation } from './DiscussionParticipation';
+import { DiscussionSearchResults } from './DiscussionSearchResults';
+import { DiscussionSearchSortBar } from './DiscussionSearchSortBar';
+import { DiscussionThread } from './DiscussionThread';
+import { useDiscussion } from './discussion-hook';
+import { calculateParticipation } from './participation-util';
+
+interface Props {
+  sectionSlug: string;
+  resourceId: number;
+  model: DirectedDiscussionActivitySchema;
+}
+
+export const DirectedDiscussion: React.FC<Props> = ({ sectionSlug, resourceId, model }) => {
+  const [searchTerm, setSearchTerm] = React.useState('');
+  const [focusId, setFocusId] = React.useState<number | null>(null);
+
+  const { loading, posts, addPost, currentUserId, deletePost } = useDiscussion(
+    sectionSlug,
+    resourceId,
+  );
+
+  const currentParticipation = useMemo(
+    () => calculateParticipation(model.participation, posts, currentUserId),
+    [posts, currentUserId],
+  );
+
+  if (!currentUserId) {
+    return null;
+  }
+
+  const onSearch = (search: string) => {
+    setSearchTerm(search);
+    setFocusId(null);
+  };
+
+  const hasSearchTerm = !!searchTerm && searchTerm.length > 0;
+  const hasFocusId = focusId !== null;
+
+  const displaySearchResults = !hasFocusId && hasSearchTerm;
+  const displayThreads = !displaySearchResults;
+
+  return (
+    <div className="activity mc-activity">
+      <div className="activity-content relative">
+        <h2>Discussion</h2>
```

Comment:
I think we should remove this `<h2>Discussion</h2>` as it hardcodes in a label that will not be changeable by an author.  The author can add a header within the stem which labels the discussion any way that they would like.